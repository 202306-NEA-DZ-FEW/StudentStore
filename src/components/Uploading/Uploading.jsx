import React from "react";

const Uploading = ({ text }) => {
    return (
        <div className='flex  items-center flex-col justify-center h-[448px] '>
            <span className='loading loading-bars loading-lg text-orange-500'></span>
            <p className='font-bold text-gray-700'>{text}</p>
        </div>
    );
};

export default Uploading;
