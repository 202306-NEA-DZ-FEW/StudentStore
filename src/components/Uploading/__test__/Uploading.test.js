import renderer from "react-test-renderer";

import Uploading from "../Uploading";

it("renders correctly", () => {
    const tree = renderer.create(<Uploading />).toJSON();
    expect(tree).toMatchSnapshot();
});
