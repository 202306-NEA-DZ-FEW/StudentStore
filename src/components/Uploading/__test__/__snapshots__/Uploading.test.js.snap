// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex  items-center flex-col justify-center h-[448px] "
>
  <span
    className="loading loading-bars loading-lg text-orange-500"
  />
  <p
    className="font-bold text-gray-700"
  />
</div>
`;
